### PR TITLE
Service split

### PR DESCRIPTION
**Pull request by mmadariaga**
Slit services so that there are no persist operations out of their entity service folder

#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX)

### CODE REVIEW

**Comment by Kaian** [comment]
This seems to require a new rebase, some changes belong to #1136

File: library/Ivoz/Provider/Domain/Service/BalanceMovement/CreateByCompany.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    protected $entityTools;
+
+    public function __construct(
+        EntityTools $entityTools
+    ) {
+        $this->entityTools = $entityTools;
+    }
+
+    public function execute(
+        CompanyInterface $carrier,
```
Comment (Kaian) [inline library/Ivoz/Provider/Domain/Service/BalanceMovement/CreateByCompany.php]: `CompanyInterface $company` here

File: library/Ivoz/Provider/Domain/Service/CallCsvScheduler/UpdateLastExecutionDate.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+{
+    /**
+     * @var EntityTools
+     */
+    private $entityTools;
+
+    /**
+     * @var LoggerInterface
+     */
+    protected $logger;
```
Comment (Kaian) [inline library/Ivoz/Provider/Domain/Service/CallCsvScheduler/UpdateLastExecutionDate.php]: I think we now remove these commentaries

**Comment by ironArt3mis** [comment]
Rebase completed.

File: library/Ivoz/Provider/Domain/Service/TransformationRuleSet/DisableGenerateRules.php [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+use Ivoz\Core\Application\Service\EntityTools;
+use Ivoz\Provider\Domain\Model\TransformationRuleSet\TransformationRuleSetDto;
+use Ivoz\Provider\Domain\Model\TransformationRuleSet\TransformationRuleSetInterface;
+
+class DisableGenerateRules
+{
+    /**
+     * @var EntityTools
+     */
+    protected $entityTools;
```
Comment (Kaian) [inline library/Ivoz/Provider/Domain/Service/TransformationRuleSet/DisableGenerateRules.php]: I think we now remove these commentaries

**Pull request merged**